src/jsMain/kotlin/com/adamratzman/spotify/utils/Types.kt

```diff
@@ -7,4 +7,4 @@ actual typealias BufferedImage = File
 
 actual typealias ConcurrentHashMap<K, V> = HashMap<K, V>
 
-actual fun <K, V> ConcurrentHashMap<K, V>.toList(): List<Pair<K, V>> = toList()
+actual fun <K, V> ConcurrentHashMap<K, V>.toList(): List<Pair<K, V>> = entries.map { (key, value) -> key to value }
```

Comment:
Why this instead of `toList()`?

Reply:
Maybe `toList()` does work but it shows a recursion arrow in the IDE and to understand why it works you need to know that extension functions have a lower priority than class functions. I think it's more understandable this way.